Makefile

```
@@ -2,6 +2,13 @@ NAME       := mpifileutils
 SRC_EXT    := gz
 REPO_NAME  := mpifileutils-pkg
 
+GIT_SHORT       := $(shell git rev-parse --short=8 HEAD)
+GIT_NUM_COMMITS := $(shell git rev-list HEAD --count)
+
+BUILD_DEFINES   := --define "relval .$(GIT_NUM_COMMITS).g$(GIT_SHORT)"
```

My apologies.  DAOS was in fact a bad example since it's something that builds from it's own source tree.

The above should be quite simply:

```suggestion
SRC_EXT   := gz
REPO_NAME := mpifileutils-pkg		￼REPO_NAME  := mpifileutils-pkg
GIT_SHORT := 7c32b9c

BUILD_DEFINES := --define "relval g$(GIT_SHORT)"
```

There is likely some automation around that even, if/once we standardise on a format for the "catch up" patch file.

But for now, hard-coding as above would be the way forward.

---

Sure, that makes sense 👍 
One more question...
--short=8 would give "7c32b9cf"
But the git default short only gives 7 characters: "7c32b9c"
Should I hardcode 7 or 8?

---

Match whatever is in the patch name, which I think is 7, probably because that is the default.

Not sure why we have `--short=8` in there.  I think it was matching something specific, but no need for it here.

---

Yes, the patch name is 7. Just pushed. Thanks for the feedback/help!
I'll let my DAOS PR run against these artifacts, and I'll check back once everything looks good.